docs/api/batch.md

```diff
@@ -194,7 +194,7 @@ return a 200 status code, and provide per-object errors. Here is an example:
   "objects": [
     {
       "oid": "1111111",
-      "size": 123,
+      "size": 123
```

Comment:
I don't think this is correct.  There's another entry after this, so we need the comma.